### ISSUE TITLE
闪卡学习数量显示规则 Flash card learning number display rules

### ISSUE THREAD

**Issue by Benjamin-Watson0** [siyuan-note/siyuan#8305]
### In what scenarios do you need this feature?

I created three new cards

Flashcard 0

Flashcard 1

Flashcard 2
![image](https://github.com/siyuan-note/siyuan/assets/112467275/6788955c-05af-426c-8256-4739257d9d24)

A card is clicked and forgotten

![image](https://github.com/siyuan-note/siyuan/assets/112467275/1b311b37-94da-4efe-a8ee-b78292f83617)

A card clicked Simple

![image](https://github.com/siyuan-note/siyuan/assets/112467275/df2194c8-9a00-41ff-8860-4ff057399326)

I can understand the meaning of a new card, that is, learning one less card.

I can understand the meaning of total cards, that is, how many cards there are in total

Why are there three review cards without learning?

I still want to illustrate it with anki's style

![image](https://github.com/siyuan-note/siyuan/assets/112467275/3d36728f-7403-48d6-ae42-e5109b730788)

There is no doubt that the unlearned is the new card.

![image](https://github.com/siyuan-note/siyuan/assets/112467275/b806a9fa-ae4e-412f-bc95-ea687aa46e94)

Learned is the card being learned

A card has two directions

Click on easy, hard or complex, as long as these three options are shown not today, you will enter the review phase when you click on them, if the time shown is today (one minute, two minutes...) , then it will enter the learning phase.

If the card enters the learning phase, its next step is to enter the review phase. In layman's terms, it keeps learning until it is easy, hard, or complicated, and as long as the three options shown are not today, it will enter the review phase.
If you click on easy, generally speaking, the time shown is not today's time, and you will go directly to the review phase, which will be shown in the due card in the future.
![try](https://github.com/siyuan-note/siyuan/assets/112467275/e6a619c8-c217-4951-95e9-cd7bb657b56e)

Siyuan didn't understand this logic
And yet it ignores the learning phase of flashcards, I don't know how many went into the learning phase. It's like trying to memorize 100 words, you didn't record the number you just memorized, I memorized a few minutes trying to see how many words I just memorized, I don't know.
Showing the number of cards in the learning stage is telling you how many words I just memorized. How many more cards in the learning stage are not yet in the review stage.

This piece of content was written by scratching the air, so maybe there are some settings I didn't see.

---

### In what scenarios do you need this feature?

我新建了三张卡片

闪卡0

闪卡1

闪卡2
![image](https://github.com/siyuan-note/siyuan/assets/112467275/6788955c-05af-426c-8256-4739257d9d24)

一张卡片点了忘记

![image](https://github.com/siyuan-note/siyuan/assets/112467275/1b311b37-94da-4efe-a8ee-b78292f83617)

一张卡片点击了简单

![image](https://github.com/siyuan-note/siyuan/assets/112467275/df2194c8-9a00-41ff-8860-4ff057399326)

新卡片的意思我可理解，就是学一张少一张。

总卡片的意思我可理解，就是总共有多少张

这复习卡片为什么没学就有三张呢

我还是想举anki的样式来说明

![image](https://github.com/siyuan-note/siyuan/assets/112467275/3d36728f-7403-48d6-ae42-e5109b730788)

未学习也就是新卡片，这点没疑问。

![image](https://github.com/siyuan-note/siyuan/assets/112467275/b806a9fa-ae4e-412f-bc95-ea687aa46e94)

学习就是学习中的卡片

一张卡片有两个走向

点击简单、困难、复杂，只要这三个选项中显示的不是今天，你点击后都会进入复习阶段，如果显示的是今天的时间（一分钟，两分钟。。。），则会进入学习阶段。

如果卡片进入学习阶段，它下一步就是进入复习阶段。通俗讲，就是一直学习，直到简单、困难、复杂，只要这三个选项中显示的不是今天，就会进入复习阶段。
点击简单，通常来讲显示的都不是今天的时间，会直接进入复习阶段，将来会在到期卡里显示。
![try](https://github.com/siyuan-note/siyuan/assets/112467275/e6a619c8-c217-4951-95e9-cd7bb657b56e)

思源这个逻辑没看懂
而且却忽略了闪卡的学习阶段，我不知道有多少进入了学习阶段。这就像想背100单词，你没记录刚才背的数量，我背了几分钟想看了我刚才背了多少单词，我不知道。
展示学习阶段卡片的数量就是告诉你，刚才我背了多少单词。还有多少学习中的卡片未进入复习阶段。

这篇内容是抓空写的，也许有些设置我没看到。

### Describe the optimal solution

I created three new cards.
Flashcard 0

Flashcard 1

Flashcard 2
![image](https://github.com/siyuan-note/siyuan/assets/112467275/6788955c-05af-426c-8256-4739257d9d24)

A card is clicked and forgotten

![image](https://github.com/siyuan-note/siyuan/assets/112467275/1b311b37-94da-4efe-a8ee-b78292f83617)

A card clicked Simple

![image](https://github.com/siyuan-note/siyuan/assets/112467275/df2194c8-9a00-41ff-8860-4ff057399326)

I can understand the meaning of a new card, that is, learning one less card.

I can understand the meaning of total cards, that is, how many cards there are in total

Why are there three review cards without learning?

I still want to illustrate it with anki's style

![image](https://github.com/siyuan-note/siyuan/assets/112467275/3d36728f-7403-48d6-ae42-e5109b730788)

There is no doubt that the unlearned is the new card.

![image](https://github.com/siyuan-note/siyuan/assets/112467275/b806a9fa-ae4e-412f-bc95-ea687aa46e94)

Learned is the card being learned

A card has two directions

Click on easy, hard or complex, as long as these three options are shown not today, you will enter the review phase when you click on them, if the time shown is today (one minute, two minutes...) , then it will enter the learning phase.

If the card enters the learning phase, its next step is to enter the review phase. In layman's terms, it keeps learning until it is easy, hard, or complicated, and as long as the three options shown are not today, it will enter the review phase.
If you click on easy, generally speaking, the time shown is not today's time, and you will go directly to the review phase, which will be shown in the due card in the future.
![try](https://github.com/siyuan-note/siyuan/assets/112467275/e6a619c8-c217-4951-95e9-cd7bb657b56e)

Siyuan didn't understand this logic
And yet it ignores the learning phase of flashcards, I don't know how many went into the learning phase. It's like trying to memorize 100 words, you didn't record the number you just memorized, I memorized a few minutes trying to see how many words I just memorized, I don't know.
Showing the number of cards in the learning stage is telling you how many words I just memorized. How many more cards in the learning stage are not yet in the review stage.

This piece of content was written by scratching the air, so maybe there are some settings I didn't see.

**Comment by 88250** [comment]
复习卡指的是到期的卡片。

**Comment by 88250** [comment]
3 张卡片的例子：

1 重来，1 简单后应该是：

![image](https://github.com/siyuan-note/siyuan/assets/873584/d529bf0e-dbb7-4e1d-8d60-3d72901fd1d7)

你的截图里是 2 2 3 多半是因为重来的 1 分钟已经到了，所以中间的数字是 2。

> 我没学怎么还有到期的卡片呢😥

到期指的是已经到间隔重复时间了，就算还没有学，第一次也是到期。

**Comment by 88250** [comment]
我先关闭了，如果还有问题请继续跟帖，谢谢 :pray:

**Comment by 88250** [comment]
设计上还是保持统一为好：

![image](https://github.com/siyuan-note/siyuan/assets/873584/8ae0e8bc-4ac2-4a36-a9e2-db36328ffeb5)

**Comment by 88250** [comment]
图裂了，麻烦重新传一下。

**Comment by 88250** [comment]
当前/一共
